tube_follower_node: 
  ros__parameters:
    desired_lateral_distance: 0.0
    desired_longitudinal_distance: 3.0
    viewing_angle: 360
    degree_window_size: 15.0
    front_degree_angle: 0.0
    right_degree_angle: 90.0
    left_degree_angle: 270.0
